src/main/scala/com/fulcrumgenomics/util/UpdateGffContigNames.scala

```diff
@@ -92,7 +92,7 @@ class UpdateGffContigNames
         val position = fields(3).toInt
         out.append(targetName)
         out.append('\t')
-        out.append(line.substring(srcName.length))
+        out.append(line.substring(srcName.length + 1)) // skip over the previous target name AND trailing tab
         out.append('\n')
```

Comment:
Actually, while we are in off by one territory, does the BetterBuffereIteror strip the newline? Or are you getting 2 newlines?

Reply:
`in` comes all the way back from `Io.readLines`, which strips the newline.